tracker/subject.go

```diff
@@ -77,7 +80,34 @@ func (s Subject) SetDomainUserId(domainUserId string) {
 	s.payload.Add(DOMAIN_UID, common.NewString(domainUserId))
 }
 
+// SetDomainSessionId adds a domain session id to the key-value store.
+func (s Subject) SetDomainSessionId(domainSessionId string) {
+	s.payload.Add(DOMAIN_SID, common.NewString(domainSessionId))
+}
+
+// SetDomainSessionIndex adds a domain user id to the key-value store.
```

Comment:
The comment suggest a user id but the code says a session index?  The function and comment seem at odds!

Reply:
Good catch! Have now fixed the commnet.